plugins/catalog-backend/src/util/unindent.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For use in tagged templates, to remove initial indentation on all lines.
+ *
+ * This is convenient for specifying e.g. YAML data.
+ */
+export function unindent(
+  strings: TemplateStringsArray,
+  ...values: string[]
+): string {
+  // Interweave the strings with the substitution vars first
+  let output = '';
+  for (let i = 0; i < values.length; i++) {
+    output += strings[i] + values[i];
+  }
+  output += strings[values.length];
+
+  // Split on newlines
+  const lines = output.split(/(?:\r\n|\n|\r)/);
```

Comment:
`\r?\n`? `\r` isn't really a thing 😅

Reply:
Proudly copy pasted from the intarwebz! 😛

Reply:
Can't that be a thing on windows...

Reply:
nah, it's pre-osx Macs, windows is \r\n